subresource-loading.bs

```diff
@@ -438,15 +438,55 @@ detailed motivation.
 To <dfn>parse a web bundle string</dfn>, given a [=string=] |sourceText| and a
 [=URL=] |baseURL|:
 
-1. Let |parsed| be the result of [=parse JSON into Infra values|parsing JSON
-   into Infra values=] given |sourceText|.
-1. If |parsed| is not a [=map=], then throw a {{TypeError}} indicating that the
-   top-level value needs to be a JSON object.
-1. Create a [=web bundle parse result=] from |parsed| and |baseURL|, and return
-   the [=web bundle parse result=].
-
-ISSUE: Specify the details. Throw a {{TypeError}} etc. on a parse error.
-Relative URLs are resolved on |baseURL|.
+1.  Let |parsed| be the result of [=parse JSON into Infra values|parsing JSON
+    into Infra values=] given |sourceText|.
+1.  If |parsed| is not a [=map=], then throw a {{TypeError}} indicating that the
```

Comment:
This is different from current Chromium implementation which throws a SyntaxError, but I think this makes more sense. We should fix our implementation.

Reply:
Ack.